aggregator/src/test/java/org/jenkinsci/plugins/workflow/steps/build/BuildTriggerStepTest.java

```diff
@@ -165,4 +170,25 @@ public void cancelBuildQueue() throws Exception {
         j.assertLogContains("branch=release extra=true", ds.getBuildByNumber(3));
     }
 
+    @Issue("JENKINS-26123")
+    @Test public void noWait() throws Exception {
+        j.createFreeStyleProject("ds").setAssignedLabel(Label.get("nonexistent"));
+        WorkflowJob us = j.jenkins.createProject(WorkflowJob.class, "us");
+        us.setDefinition(new CpsFlowDefinition("build job: 'ds', wait: false"));
+        j.assertBuildStatusSuccess(us.scheduleBuild2(0));
```

Comment:
Cleared up some confusion on the phone: `QDH` is active during the `rejectedStart` test, _not_ during the `noWait` test. For the first test, `ds` is successfully added to the queue (but never builds), so `us` succeeds; for the second test, `ds` is not even added to the queue, so `us` fails.